workflows/prognostic_c48_run/runtime/segmented_run/run.py

```diff
@@ -22,8 +22,23 @@ def cwd(path):
 def find(path: str):
     return glob.glob(os.path.join(path, "**"), recursive=True)
 
+def compose_simulation_command(mpi_launcher: str, nprocs: int): 
 
-def run_segment(config: dict, rundir: str):
+    command = ["-n " + str(nprocs), sys.executable, "-m", "mpi4py", runfile.absolute().as_posix()]
+    
+    if mpi_launcher == "mpirun":
+        command.insert(0, mpi_launcher)
```

Comment:
nit: since this is needed for both the supported `mpi_launcher` options, it might make sense to just put in the `command` at the start.